ambry-admin/src/test/java/com.github.ambry.admin/AdminIntegrationTest.java

```diff
@@ -307,7 +307,7 @@ private void doGetHeadDeleteTest(int contentSize, boolean multipartPost) throws
     String contentType = "application/octet-stream";
     String ownerId = "getHeadDeleteOwnerID";
     HttpHeaders headers = new DefaultHttpHeaders();
-    setAmbryHeaders(headers, content.capacity(), 7200, false, serviceId, contentType, ownerId);
+    setAmbryHeadersForPut(headers, 7200, false, serviceId, contentType, ownerId);
     headers.set(HttpHeaderNames.CONTENT_LENGTH, content.capacity());
```

Comment:
don't remember why content length is set but this might be something you want to remove too